data-raw/simulating-DAG.R

```diff
@@ -1,5 +1,62 @@
-# A DAG
 library(dagitty)
+<<<<<<< HEAD
+library(dplyr)
+library(survival)
+################################################
+# Define data-generazing model (DAG) == g
+# including exposure and network-variables (a-p)
+g <- dagitty('dag{
+            Exposure -> b[beta=.15] Exposure -> l[beta=.15] Exposure -> o[beta=.15];
+             a -> b[beta=.3] b -> c[beta=.2] c -> d[beta=.5] d -> e[beta=.3] e -> f[beta=.6]
+             b -> k [beta=.3] k -> l [beta=.3] l -> m [beta=.5] ;
+             m -> o [beta=.3] o -> p [beta=.5];
+             a -> v [beta=.2] ;
+             c -> v [beta=.5] ;
+             d -> v [beta=.3] ;
+             e -> m [beta=.5] ;
+             k -> o [beta=.3] ;
+             o -> p [beta=.65] }')
+################################################
+# Simulate data based on the DAG g
+x <- simulateSEM(g, N=20000)
```

Comment:
better to call x by a more descriptive name, aka "simulated_metabolite_data" or something